syntax = "proto3";

package message;

// 验证服务
service VarifyService {
  rpc GetVarifyCode (GetVarifyReq) returns (GetVarifyRsp) {} // 获取验证码接口
}

// 获取验证码请求
message GetVarifyReq {
  string email = 1;
}

// 获取验证码响应
message GetVarifyRsp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}